features/test/apiserver.feature

```diff
@@ -0,0 +1,26 @@
+Feature: API server Test features
+  # @author kewang@redhat.com
+  @admin
+  @destructive
+  Scenario: Quick test for step definitions
+    Given the master version >= "4.6"
+    Given I switch to cluster admin pseudo user
+
+    # Set WriteRequestBodies profile to audit log
+    Given as admin I successfully merge patch resource "apiserver/cluster" with:
+      | {"spec": {"audit": {"profile": "WriteRequestBodies"}}} |
+    And I register clean-up steps:
+    # Set original Default profile to audti log
+    """
+    Given as admin I successfully merge patch resource "apiserver/cluster" with:
+      | {"spec": {"audit": {"profile": "Default"}}} |
+    Given operator "kube-apiserver" becomes "progressing" within 100 seconds
+    Given operator "kube-apiserver" becomes "available/non-progressing/non-degraded" within 1200 seconds
```

Comment:
@wangke19 did these 2 lines with quotes passed test? If not, could you update them?

Reply:
Fixed in PR https://github.com/openshift/verification-tests/pull/1837